types/pg-pool/index.d.ts

```diff
@@ -1,7 +1,28 @@
-// Type definitions for pg-pool
+// Type definitions for pg-pool 2.0
 // Project: https://github.com/brianc/node-pg-pool
-// Definitions by: Leo Liang <https://github.com/aleung>
+// Definitions by: Leo Liang <https://github.com/aleung>, Nikita Tokarchuk <https://github.com/mainnika>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-/// <reference types="pg" />
-export {Pool, PoolConfig } from "pg";
+import * as pg from 'pg';
+
+declare class Pool<T extends pg.Client> extends pg.Pool {
```

Comment:
Extend EventEmitter here instead.

Reply:
I can either move the entire pg.Pool types here or keep it like this.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/pg/index.d.ts#L142

Reply:
Sorry about that I didn't notice that pg.Pool does extend EventEmitter